packages/web/src/scss/tools/__tests__/_accessibility.test.scss

```diff
@@ -0,0 +1,82 @@
+// stylelint-disable scss/at-mixin-argumentless-call-parentheses -- We need to allow this to make sass-true work
+@use 'true';
```

Comment:
Can be `true` used somehow globally? Like we are not writing `true.describe` but only `describe`...?

Reply:
AFAIK not :)